plugin_tests/annotation_and_mask_utils_test.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
```

Comment:
These new test files are being run as part of the CI ctest process.  Because this repo isn't quite a simple Girder 2.x plugin, tests have to be added explicitly in the `plugin.cmake` file.  Specifically, add appropriate lines like:
```
add_histomicstk_python_test(annotation_and_mask_utils)

add_histomicstk_python_test(annotations_to_masks_handler)
```
I would add this just above the `# front-end tests` section in `plugin.cmake`.

Reply:
@manthey After these new changes, integration fails for python 2.7 for some weird reason. It says:
```
35: AttributeError: 'module' object has no attribute 'annotations_to_masks_handler_test'
35: Coverage.py warning: No data was collected. (no-data-collected)
23/39 Test #35: server_HistomicsTK.annotations_to_masks_handler ......................***Failed    0.40 sec
test 36
      Start 36: web_client_HistomicsTK.annotations
```
Any idea what's going on?

Reply:
I'm not sure yet -- I'm looking into it.

Reply:
`FileExistsError` in `annotations_to_masks_handler_test` is a Python3 exception, so it doesn't work in Python 2.  I'll fix it along with the other change I'm making.

Reply:
Ah I see. thank you :)